modules/client/src/Wallet.ts

```diff
@@ -52,8 +53,14 @@ export class Wallet extends eth.Signer {
       this.signer = eth.Wallet.fromMnemonic(opts.mnemonic || '')
       this.signer = this.signer.connect(this.provider)
       this.address = this.signer.address.toLowerCase()
-
-    // Third choice: Sign w web3
+    // Third choice: External wallets
+    } else if (opts.externalWallet) {
+      this.signer = opts.externalWallet
+      this.external = true
+      this.web3 = new Web3(opts.web3Provider as any)
```

Comment:
I'm a little confused, while using an external wallet:
 - If `signTransaction` is called, it'll ultimately be passed to `opts.externalWallet.sign()`
 - If `sendTransaction` is called, it'll end up at `web3.eth.sendTransaction()`
 - If `signMessage` is called, it'll end up at `opts.externalWallet.signMessage()`

Why use `externalWallet` interface to sign some stuff and `web3` in other cases? Could we just use one or the other?